docs/references/output.md

```diff
@@ -45,14 +46,87 @@ outputs:
 
 Here we can use {{filter.htmlname}} like in parsers and scenarios with the {{signal.htmlname}} object to choose which signal will be process by which plugin.
 
-## Plugins
 
-Plugins configuration file are stored in `{{plugins.configpath}}`. {{crowdsec.name}} will scan this folder to load all the plugins. Each configuration file should provide the path to the plugin binary. By default they are stored in `{{plugins.binpath}}`.
+
+# Switching backend database
```

Comment:
I think the title should be something more generic like "backend database configuration" no ?